docs/commands/bundle.md

```diff
@@ -1,117 +1,91 @@
-  --lint               Lint definitions               [boolean] [default: false]
-  --output, -o                                                          [string]
-  --format             Use a specific output format.
-                        [choices: "stylish", "codeframe"] [default: "codeframe"]
-  --max-problems       Reduce output to max N problems.  [number] [default: 100]
-  --ext                Bundle file extension.   [choices: "json", "yaml", "yml"]
-  --skip-rule          Ignore certain rules.                             [array]
-  --skip-preprocessor  Ignore certain preprocessors.                     [array]
-  --skip-decorator     Ignore certain decorators.                        [array]
-  --dereferenced, -d   Produce fully dereferenced bundle.              [boolean]
-  --force, -f          Produce bundle output even when errors occur.   [boolean]
-  --config             Specify path to the config file.                 [string]
-```
+:::success Tip
+To learn more about preprocessors, rules, and decorators, please refer to [this](../custom-rules.md) document.
+:::
 
-The command:
+## Usage
 
 ```bash
-openapi bundle --output <outputName> --ext <ext> [entrypoints...]
-```
-
-- `[entrypoints...]` corresponds to the name(s) of your root document(s).
-- Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
-- The `<outputName>` is your desired output filename or folder. If the folder doesn't exist, it's automatically created.
-- Use `--ext <ext>` to specify the type and extension of the output file: `.json`, `.yml` or `.yaml`.
+openapi bundle <entrypoints>...
+openapi bundle <entrypoints> [--max-problems=<n>]
+openapi bundle <entrypoints> [--lint] [--config=<path>]
+openapi bundle <entrypoints>... -o <outputName> --ext <ext>
+openapi bundle --version
+```
+
+## Options
+
+option                 | type      | required?    | default     | description
+-----------------------|:---------:|:------------:|:-----------:|------------
+`entrypoints`          | `array`   | yes          | `[]`        | Array of API root definition filenames that need to be bundled. Instead of full paths, you can use aliases assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints.
+`--config`             | `string`  | no           | -           | Specify path to the config file
+`--dereferenced`, `-d` | `boolean` | no           | -           | Generate fully dereferenced bundle
+`--ext`                | `string`  | no           | `yaml`      | Specify bundled file extension.<br />**Possible values:** `json`, `yaml`, `yml`
+`--force`, `-f`        | `boolean` | no           | -           | Generate bundle output even when errors occur
+`format`               | `string`  | no           | `codeframe` | Format for the output.<br />**Possible values:** `codeframe`, `stylish`
```

Comment:
question: What does `codeframe` do? What does `stylish` do? What's the difference between them; when to use one, and when to use the other?

Reply:
I believe you can find answers for that in the PR for the [lint command](https://github.com/Redocly/openapi-cli/pull/360)

Reply:
So the idea is that the users should look there to find that info, too? If that's the case, the least we can do is provide a link.

Reply:
Added information about undocumented `--format=json` option. `codeframe` and `stylish` left blank. If it doesn't look good, we can leave only `json` atm